Documentation/openshiftdocs/modules/clustermonitoringconfiguration.adoc

```diff
@@ -1,32 +1,35 @@
 
 === Description
 
-ClusterMonitoringConfiguration defines configuration that allows users to customise the platform monitoring stack through the cluster-monitoring-config ConfigMap in the openshift-monitoring namespace
+The `ClusterMonitoringConfiguration` resource defines settings that customize the default platform monitoring stack through the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace.
 
 
 [options="header"]
 |===
 | Property | Type | Description 
-|alertmanagerMain|*link:alertmanagermainconfig.adoc[AlertmanagerMainConfig]|AlertmanagerMainConfig defines configuration related with the main Alertmanager instance.
+|alertmanagerMain|*link:alertmanagermainconfig.adoc[AlertmanagerMainConfig]|The `AlertmanagerMainConfig` resource defines settings for the Alertmanager component in the `openshift-monitoring` namespace.
 
-|enableUserWorkload|*bool|UserWorkloadEnabled boolean flag to enable monitoring for user-defined projects.
+|enableUserWorkload|*bool|\n `UserWorkloadEnabled` a Boolean flag that enables monitoring for user-defined projects.
 
-|k8sPrometheusAdapter|*link:k8sprometheusadapter.adoc[K8sPrometheusAdapter]|K8sPrometheusAdapter defines configuration related with prometheus-adapter
+|k8sPrometheusAdapter|*link:k8sprometheusadapter.adoc[K8sPrometheusAdapter]|`K8sPrometheusAdapter` defines settings for the Prometheus Adapter component.
 
-|kubeStateMetrics|*link:kubestatemetricsconfig.adoc[KubeStateMetricsConfig]|KubeStateMetricsConfig defines configuration related with kube-state-metrics agent
+|kubeStateMetrics|*link:kubestatemetricsconfig.adoc[KubeStateMetricsConfig]|`KubeStateMetricsConfig` defines settings for the `kube-state-metrics` agent.
 
-|prometheusK8s|*link:prometheusk8sconfig.adoc[PrometheusK8sConfig]|PrometheusK8sConfig defines configuration related with prometheus
+|prometheusK8s|*link:prometheusk8sconfig.adoc[PrometheusK8sConfig]|`PrometheusK8sConfig` defines settings for the Prometheus component.
 
-|prometheusOperator|*link:prometheusoperatorconfig.adoc[PrometheusOperatorConfig]|PrometheusOperatorConfig defines configuration related with prometheus-operator
+|prometheusOperator|*link:prometheusoperatorconfig.adoc[PrometheusOperatorConfig]|`PrometheusOperatorConfig` defines settings for the Prometheus Operator component.
 
-|openshiftStateMetrics|*link:openshiftstatemetricsconfig.adoc[OpenShiftStateMetricsConfig]|OpenShiftMetricsConfig defines configuration related with openshift-state-metrics agent
+|openshiftStateMetrics|*link:openshiftstatemetricsconfig.adoc[OpenShiftStateMetricsConfig]|`OpenShiftMetricsConfig` defines settings for the `openshift-state-metrics` agent.
 
-|thanosQuerier|*link:thanosquerierconfig.adoc[ThanosQuerierConfig]|ThanosQuerierConfig defines configuration related with the Thanos Querier component
+|thanosQuerier|*link:thanosquerierconfig.adoc[ThanosQuerierConfig]|`ThanosQuerierConfig` defines settings for the Thanos Querier component.
 
 |===
```

Comment:
no telemeterClient, don't we need to mention it? @simonpasquier

Reply:
Yes I agree that it should be documented. Not sure why it was omitted in 1495d3bde. Maybe @JoaoBraveCoding would remember?

Reply:
I vaguely remember that some fields we wanted to omit some since they shouldn't be configured by users, and if I'm remembering correctly telemeterClient was part of that list... but I'm not finding that information.... though it was on the issue https://issues.redhat.com/browse/MON-1615 but it's not...
Regardless if we want to have it now documented we should only need to remove the comment `// OmitFromDoc`

Reply:
Ok. Thanks. I've deleted `// OmitFromDoc` and added:

```
// `TelemeterClientConfig` defines settings for the Telemeter Client
// component.

```